package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonExporter.cs

```diff
@@ -80,6 +80,8 @@ public static class SkeletonExporter {
       // This is required because USD expects the extent to be a local bound for the SkelRoot.
       var oldParent = objContext.gameObject.transform.parent;
       objContext.gameObject.transform.SetParent(null, worldPositionStays: false);
+      var originalScale = objContext.gameObject.transform.localScale;
+      objContext.gameObject.transform.localScale *= 1 / exportContext.scale;
```

Comment:
I do believe this was intended to be capture by `SetParent(null, ...)` above, it's just that the scale wasn't captured.

Reply:
I guess my remaining question is if this should be `1/exportContext.scale` or if it should just be `1`.